tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -66,3 +66,86 @@ def __str__(self):
     def _get_error_message(self):
         """Return the error message."""
         return self.error_message
+
+class DataFileSummary(models.Model):
+    """Aggregates information about a parsed file."""
+
+    class Status(models.TextChoices):
+        """Enum for status of parsed file."""
+
+        PENDING = "Pending"  # file has been uploaded, but not validated
+        ACCEPTED = "Accepted"
+        ACCEPTED_WITH_ERRORS = "Accepted with Errors"
+        REJECTED = "Rejected"
+
+    status = models.CharField(
+        max_length=50,
+        choices=Status.choices,
+        default=Status.PENDING,
+    )
+
+    datafile = models.ForeignKey(DataFile, on_delete=models.CASCADE)
+
+    #  eventually needs a breakdown of cases (accepted, rejected, total) per month
+    #  elif qtr2 jan-mar
+    #  elif qtr3 apr-jun
+    #  elif qtr4 jul-sept
+
+    case_aggregates = models.JSONField(null=True, blank=False)
+    """
+    # Do these queries only once, save result during creation of this model
+    # or do we grab this data during parsing and bubble it up during create call?
+    {
```

Comment:
This sounds like a signal to a separate model, so whenever there is a new DataFileSummary, then the aggregate function runs based on the Datafile.date and adds the number to that new model.

Reply:
Right now this field is just a placeholder but it gets implemented in our .2 PR. Perhaps this comment still applies there? [Check `def case_aggregates_by_month()` out] and let me know.